bloom-instance/tfvars.template

```diff
@@ -3,4 +3,32 @@ application_name = "Bloom Housing Instance"
 resource_prefix = "doorway_bloom"
 team_name = "doorway"
 aws_region = "us-west-1"
-sdlc_stage = "dev"
\ No newline at end of file
+sdlc_stage = "dev"
+subnet_map = {
+    "public" = [
+      {
+        az   = "${var.aws_region}a", # us-west-1a
```

Comment:
The tfvars.template file is absolutely the correct place to put this, but unfortunately you can't reference other vars like aws_region in a tfvars file.  Rather than interpolating var.aws_region here, just go ahead and add the values as "us-west-1a", etc instead.